### PR TITLE
Add function to convert hermes maps to gammapy compatible format

### PR DESCRIPTION
**Pull request by QRemy**
Add a function to convert hermes maps (https://github.com/cosmicrays/hermes) to a gammapy compatible format

### CODE REVIEW

File: gammapy/modeling/models/utils.py [hunk 1]
```diff
@@ -46,3 +47,38 @@ def _template_model_from_cta_sdc(filename):
...
+    energy_nodes = []
+    with fits.open(filename) as hdulist:
+        # cannot read directly in 3d with Map.read because BANDS HDU is missing
+        # https://gamma-astro-data-formats.readthedocs.io/en/v0.2/skymaps/index.html#bands-hdu
+        # so we have to loop over hdus and create the energy axis
+        for hdu in hdulist[1:]:
+            template = HpxNDMap.from_hdu(hdu, format="hermes-template")
+            # fix missing/incompatible infos
+            template._unit = u.Unit(hdu.header["TUNIT1"])  # .from_hdu expect "BUNIT"
+            if template.geom.frame == "G":
```
Comment (AtreyeeS) [inline gammapy/modeling/models/utils.py:74]: No option required for ICRS?

File: gammapy/modeling/models/utils.py [hunk 2]
```diff
@@ -46,3 +47,38 @@ def _template_model_from_cta_sdc(filename):
...
+                template._geom._frame = "galactic"
+            maps.append(template)
+            energy_nodes.append(hdu.header["ENERGY"])  # SI unit (see header comment)
+    # create energy axis and set unit
+    energy_nodes *= u.Joule
```
Comment (AtreyeeS) [inline gammapy/modeling/models/utils.py:79]: hermes energies are always in joules?

**Review by AtreyeeS — APPROVED** [review]
Thanks @QRemy. I don't know if you can add a link to the Hermes format definition somewhere.

**Comment by QRemy** [comment]
As far as I know hermes maps are always produces with the same units and format, maybe @carmeloevoli could confirm ?

**Review by Astro-Kirsty — APPROVED** [review]
I agree with the comments by @AtreyeeS. 
Otherwise all good, thanks @QRemy

**Comment by carmeloevoli** [comment]
> As far as I know hermes maps are always produced with the same units and format, maybe @carmeloevoli could confirm ?

Yep, all gamma maps have the same units and format.

**Pull request merged**